src/services/query.ts

```diff
@@ -65,29 +65,32 @@ export class QueryTermsService {
 
   async run(): Promise<QueryResult> {
     this.logger.info(
-      `Querying "${this.accessService.endpointUrl}" with "${this.query}"...`
+      `Querying "${this.dataset.distribution.url}" with "${this.query}"...`
     );
     const config = this.getConfig();
     const timer = new Hoek.Bench();
     const result = (await this.engine.query(
-      this.accessService.query,
+      this.dataset.distribution.query,
       config
     )) as IActorQueryOperationOutputQuads;
 
     return new Promise((resolve, reject) => {
       const termsTransformer = new TermsTransformer();
-      result.quadStream.on('error', reject);
+      result.quadStream.on('error', error => {
+        console.log('error', error);
```

Comment:
Is it necessary to log the error here? Can we leave it to the caller and simply reject instead?